crates/interpreter/src/side_table.rs

```diff
@@ -33,12 +33,12 @@ pub struct SideTableEntryView {
 
 #[allow(dead_code)] // TODO(dev/fast-interp)
 impl SideTableEntry {
-    const DELTA_IP_MASK: u32 = 0x0000ffff;
-    const DELTA_STP_MASK: u32 = 0x003f0000;
-    const VAL_CNT_MASK: u32 = 0x07c00000;
-    const POP_CNT_MASK: u32 = 0xf8000000;
+    const DELTA_IP_MASK: u32 = 0x7ff;
+    const DELTA_STP_MASK: u32 = 0x7ffff800; // 0x7fff << 11
```

Comment:
1) These two masks have been experimented to be the smallest ones to make the test suite pass. The two `.wast` file that requires much larger `DELTA_STP_MASK` are `align.wast` and `br_table.wast` (e.g. [this function](https://github.com/WebAssembly/spec/blob/c280b356e0da830ce0b7c636c34ea684f5d1b728/test/core/br_table.wast#L107)).  We may want to skip some unusual functions in tests to keep using `u32`, but it does not seem straightforward given the current setup.

2) Because of 1), the number of remaining bits for `VAL_CNT_MASK` and `POP_CNT_MASK` will likely be not enough when we compute `val_cnt` and `pop_cnt`. How about using `u64` instead if we have to keep all the test cases?